cabby/geo/sample_poi.py

```diff
@@ -0,0 +1,67 @@
+# coding=utf-8
+# Copyright 2020 Google LLC
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+'''Command line application to sample an end and starting point, the route  
+between them and pivots.
+
+Example:
+$ bazel-bin/cabby/geo/sample_poi 
+--region Manhattan --min_s2_level 18 --directory "/mnt/hackney/data/cabby/poi/
+v1/" 
+--path "/mnt/hackney/data/cabby/poi/geo_paths.gpkg" --n_samples 2
```

Comment:
Please reformat so it can be copied-and-pasted to work on the command line, e.g.:

```
$ bazel-bin/cabby/geo/sample_poi \
  --region Manhattan --min_s2_level 18 --directory "/path/to/data/"  \
  --path "/mnt/hackney/data/cabby/poi/geo_paths.gpkg" --n_samples 2
```

Note: please don't include our own machine directories locations in code!

Reply:
removed the machine directories